File: benchmarks/fair/arrayiter.rs
fn main(){
    let n:usize=std::env::var("BENCH_N").ok().and_then(|v|v.parse().ok()).filter(|&v|v>0).unwrap_or(5000000);
    let mut a=vec![0i64;n]; let mut t:i64=0;
    for i in 0..a.len() { a[i]=i as i64; }
    for i in 0..a.len() { t+=a[i]; }
    println!("{}",t);
}

File: benchmarks/fair/arrayiter.js
const n = Number(process.env.BENCH_N) > 0 ? Number(process.env.BENCH_N) : 5000000;
const a = new Float64Array(n);
let t = 0;
for (let i = 0; i < a.length; i++) a[i] = i;
for (let i = 0; i < a.length; i++) t += a[i];
console.log(String(t));
